src/tests/ftest/util/server_utils.py

```diff
@@ -204,9 +207,9 @@ def run_server(hostfile, setname, basepath, uri_path=None, env_dict=None,
         # Run server in insecure mode until Certificate tests are in place
         server_cmd.extend([daos_srv_bin,
                            "--debug",
-                           "--config", '{}/{}'.format(basepath, AVOCADO_FILE),
+                           "--config", '{}/{}'.format(build_vars["PREFIX"], AVOCADO_FILE),
                            "start", "-i",
-                           "-a", os.path.join(basepath, "install", "tmp")])
+                           "-a", os.path.join(build_vars["PREFIX"], "tmp")])
 
```

Comment:
I would not use build_vars["PREFIX"] to locate tmp.    Currently we need this to be a shared directory and in rpms this is not.  (/usr/tmp)     If this could  be an ENV then the user would be able to create their own shared dir.    
This will not be necessary once we have systemctl for daos_server but it will not work for rpms as written right now.

Reply:
Actually `build_vars["PREFIX"]` is still the shared directory in CI.

But the point of being able to set that to some other value in environments outside of our CI is important.

Reply:
So we can used build_vars["TMPDIR"]? one question is what's the default value if TMPDIR is not set by user? and which file should this env be added to?

Reply:
`build_vars["TMPDIR"]` doesn't exist.

Reply:
right, either we add TMPDIR in now if we want to solve it in this PR, or leave the user configured tmp dir to another later ticket.

Reply:
So, _TMPDIR_ is not really what this is.  It's "SHAREDDIR" or something like that.  For CI testing, it can be `build_vars["PREFIX"]`.  That will be a shared dir.  But we need to also support a mechanism for non-CI users to set this dir.  Maybe non-CI users just edit the build_vars.json to point at their shared dir.  Maybe some other mechanism.

Reply:
Tried to add a TMPDIR to daos/.build_vars.json:
  "PREFIX": "/home/huakuang/daos/install", 
  "TMPDIR": "/home/huakuang",

Then modified use it in server_utils.py here:
                           os.path.join(build_vars["TMPDIR"], "tmp")])

Launch tests from install/lib/daos/TESTING/ftest worked fine.
 ./launch.py -c -a -r -ts boro-58 conthandle

Reply:
Allow user to set TMPDIR, if  not set, default it to PREFIX